lib/link_thumbnailer/models/body.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'link_thumbnailer/model'
+
+module LinkThumbnailer
+  module Models
+    class Body < ::LinkThumbnailer::Model
+
+      attr_reader :node, :text
+
+      def initialize(node, text = nil)
+        @node = node
+        @text = sanitize(squish(node.map(&:text).join(' ')))
```

Comment:
Personally I'd prefer if it wouldn't concatenate all found paragraphs, as that will prevent proper segmentation into sentences. Instead I suggest returning an array of strings.